demoproviders/trapdoor/java/org/openyolo/demoprovider/trapdoor/CredentialQueryReceiver.java

```diff
@@ -67,7 +65,7 @@ protected void processCredentialRequest(
 
         Intent retrieveIntent = RetrieveActivity.createIntent(applicationContext);
         CredentialRetrieveBbqResponse response = CredentialRetrieveBbqResponse.newBuilder()
-                .setRetrieveIntent(ByteString.copyFrom(IntentUtil.toBytes(retrieveIntent)))
```

Comment:
So ` Intent retrieveIntent = RetrieveActivity.createIntent(applicationContext);` is not used anymore?